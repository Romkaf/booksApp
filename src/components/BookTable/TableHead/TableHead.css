.root {
	display: grid;
	grid-template-columns: repeat(auto-fit, minmax(50px, 1fr));
	border-bottom: 2px solid var(--grey);
}

.ceil {
	padding: 0.75rem;
}

@media (max-width: 500px) {
	.ceil {
		padding: 0.35rem 0.25rem;
		font-size: 16px;
	}
}
