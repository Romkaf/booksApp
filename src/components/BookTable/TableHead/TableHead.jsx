import React, { memo } from 'react';
import styles from './TableHead.css';

const TableHead = () => {
	const { root, ceil } = styles;
	const labels = ['Title', 'Authors', 'Publisher', 'Date'];

	return (
		<div className={root}>
			{labels.map((label) => (
				<div className={ceil} key={label}>
					{label}
				</div>
			))}
		</div>
	);
};

export default memo(TableHead);
