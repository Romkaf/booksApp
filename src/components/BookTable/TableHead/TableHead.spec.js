import React from 'react';
import TableHead from './TableHead';
import { shallow } from 'enzyme';

describe('TableRow', () => {
	it('should render component', () => {
		const component = shallow(<TableHead />);
		expect(component).toMatchSnapshot();
	});
});
