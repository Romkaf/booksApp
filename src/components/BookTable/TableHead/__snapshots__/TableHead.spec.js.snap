// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRow should render component 1`] = `
<div
  className="root"
>
  <div
    className="ceil"
    key="Title"
  >
    Title
  </div>
  <div
    className="ceil"
    key="Authors"
  >
    Authors
  </div>
  <div
    className="ceil"
    key="Publisher"
  >
    Publisher
  </div>
  <div
    className="ceil"
    key="Date"
  >
    Date
  </div>
</div>
`;
